data/mods/MindOverMatter/mutations/traits.json

```diff
@@ -21,6 +21,18 @@
     "valid": false,
     "spells_learned": [ [ "clair_night_vision", 2 ], [ "clair_danger_sense", 2 ], [ "clair_speed_reading", 2 ] ]
   },
+  {
+    "type": "mutation",
+    "id": "PHOTOKINETIC",
+    "name": "Awakened Photokinetic",
+    "points": 0,
+    "description": "You have the power to control the light around you with your mind.",
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "flags": [ "PHOTOKINETIC" ],
+    "spells_learned": [ [ "photokinetic_snuff_light", 1 ], [ "photokinetic_light_dodge", 1 ], [ "photokinetic_light_beam", 1 ] ]
```

Comment:
Oh yeah! I was wondering why the starting powers are given at level 2, if you dont mind explaining it to me, will change it regardless to match with the rest.

Reply:
I just did that so that the starting powers are a bit more likely to succeed, since (unlike Magiclysm) there's no way to learn later powers without having to use the earlier powers a lot.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/69092/commits/5051940ece560b4beaf623283ad85693b9d39d99
fixed here.